version-compatibility-matrix.yaml

```diff
@@ -0,0 +1,70 @@
+- meshName: "Istio"
+  versionRange: 
+    - meshVersion: "1.12"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.42.0"
+      kialiMaximumVersion: ""
+    - meshVersion: "1.11"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.38.1"
+      kialiMaximumVersion: "1.41"
+    - meshVersion: "1.10"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.34.1"
+      kialiMaximumVersion: "1.37"
+    - meshVersion: "1.9"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.29.1"
+      kialiMaximumVersion: "1.33"
+    - meshVersion: "1.8"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.26.0"
+      kialiMaximumVersion: "1.28"
+    - meshVersion: "1.7"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.22.1"
+      kialiMaximumVersion: "1.25"
+    - meshVersion: "1.6"
+      SMCPVersion: ""
+      kialiMinimumVersion: "1.18.1"
+      kialiMaximumVersion: "1.21"
```

Comment:
Just to be complete, I think this should go all the way back to meshVersion 1.0 with a maxKiali of 1.17.    But really none of this matrix will be useful because the check won't apply for kiali versions before the one that merges in this code, unless we do some backports and releases of older Kiali lines.

Reply:
But if meshVersion is 1.0, what is the minimum Version of kiali, which is compatible?

Reply:
what @jshaughn proposes seems reasonable.